fmriprep/workflows/bold.py

```diff
@@ -84,18 +92,19 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
     # Build workflow
     workflow = pe.Workflow(name=name)
     inputnode = pe.Node(niu.IdentityInterface(
-        fields=['epi', 't1_preproc', 't1_brain', 't1_mask', 't1_seg', 't1_tpms',
+        fields=['bold', 't1_preproc', 't1_brain', 't1_mask', 't1_seg', 't1_tpms',
                 't1_2_mni_forward_transform', 't1_2_mni_reverse_transform',
                 'subjects_dir', 'subject_id', 'fs_2_t1_transform']),
         name='inputnode')
-    inputnode.inputs.epi = bold_file
+    inputnode.inputs.bold = bold_file
```

Comment:
How about making `'bold'` `'bold_file'`? I think that will help make it clear that it's the `bold_file` that's passed to this function, and that `name_source` (which is what it's often passed to) is looking at the original BOLD file name.